src/Dfc.CourseDirectory.Functions/Dfc.CourseDirectory.Functions.csproj

```diff
@@ -5,7 +5,7 @@
     <UserSecretsId>CourseDirectoryFunctions</UserSecretsId>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Functions.Extensions" Version="1.0.0" />
+    <PackageReference Include="Microsoft.Azure.Functions.Extensions" Version="1.1.0-preview1" />
```

Comment:
Did you want a preview package here? If it's intentional it could do with a note in the commit message.

Reply:
Package has RTMed now - updated reference